content/zh-cn/blog/_posts/2021-07-26-update-with-ingress-nginx.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: blog
+title: '更新 NGINX-Ingress 以使用稳定的 Ingress API'
+date: 2021-07-26
+slug: update-with-ingress-nginx
+---
+<!--
+layout: blog
+title: 'Updating NGINX-Ingress to use the stable Ingress API'
+date: 2021-07-26
+slug: update-with-ingress-nginx
+-->
+
+<!--
+**Authors:** James Strong, Ricardo Katz
+-->
+**作者：** James Strong, Ricardo Katz
+
+<!--
+With all Kubernetes APIs, there is a process to creating, maintaining, and
+ultimately deprecating them once they become GA. The networking.k8s.io API group is no
+different. The upcoming Kubernetes 1.22 release will remove several deprecated APIs
+that are relevant to networking:
+-->
+对于所有 Kubernetes API，一旦它们被正式发布（GA），就有一个创建、维护和最终弃用它们的过程。
+network.k8s.io API 组也不例外。
+即将发布的 Kubernetes 1.22 版本将删除几个与网络相关的已弃用 API：
+
+<!--
+- the `networking.k8s.io/v1beta1` API version of [IngressClass](/docs/concepts/services-networking/ingress/#ingress-class)
+- all beta versions of [Ingress](/docs/concepts/services-networking/ingress/): `extensions/v1beta1` and `networking.k8s.io/v1beta1`
+-->
+- [IngressClass](/zh-cn/docs/concepts/services-networking/ingress/#ingress-class) 的 `networking.k8s.io/v1beta1` API 版本
+- [Ingress](/zh-cn/docs/concepts/services-networking/ingress/) 的所有 beta 版本: `extensions/v1beta1` 和 `networking.k8s.io/v1beta1`
+
+<!--
+On a v1.22 Kubernetes cluster, you'll be able to access Ingress and IngressClass
+objects through the stable (v1) APIs, but access via their beta APIs won't be possible.
+-->
+在 v1.22 Kubernetes 集群上，你能够通过稳定（v1）API 访问 Ingress 和 IngressClass 对象，
+但无法通过其 Beta API 访问。
+<!--
```

Comment:
Add a newline between paragraphs.

Reply:
They are in one paragraph, it is the same pattern with the en docs